docs/node-operators/get-started/setup-non-validator.md

```diff
@@ -103,30 +94,29 @@ This creates a set of configuration files which you then need to alter for your
 ```
 ## Request peers
 
-7. Build a list of Tendermint peers by reaching out to existing node operators as they can provide the data, on the [Vega Discord](https://vega.xyz/discord) for example. You'll need to format the peers data as a comma separated list.
+7. Build a list of CometBFT peers by reaching out to existing node operators as they can provide the data. You'll need to format the peers data as a comma separated list.
 
 8. Open the config file `$TENDERMINT_PATH/config/config.toml` and update the value `[p2p] -> persistent_peers` with the list created above
 
 9. Set the following values:
 
 ```toml
   [p2p]
-    max_packet_msg_payload_size=16384
     pex = true
-    seeds = "b0db58f5651c85385f588bd5238b42bedbe57073@13.125.55.240:26656,abe207dae9367995526812d42207aeab73fd6418@18.158.4.175:26656,198ecd046ebb9da0fc5a3270ee9a1aeef57a76ff@144.76.105.240:26656,211e435c2162aedb6d687409d5d7f67399d198a9@65.21.60.252:26656,c5b11e1d819115c4f3974d14f76269e802f3417b@34.88.191.54:26656,61051c21f083ee30c835a34a0c17c5d1ceef3c62@51.178.75.45:26656,b0db58f5651c85385f588bd5238b42bedbe57073@18.192.52.234:26656,36a2ca7bb6a50427be2181c8ebb7f62ac62ebaf5@m2.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,32d7380b195c088c0605c5d24bcf15ff1dade05f@api1.vega.community:26656,4f26ec99d3cf6f0e9e973c0a5f3da87d89ec6677@api2.vega.community:26656,eafacd11af53cd9fb2a14eada53485779cbee4ab@api3.vega.community:26656"
+    seeds = "insert-network-seeds-here"

```

Comment:
```suggestion
    persistent_peers = "insert-network-peers-here"
```

Let's change it to the persistent peers, as according to the documentation persistent peers also participate in peers exchange when the `pex` is set to true.